src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,145 @@
+package com.airlines.dao;
+
+import com.airlines.entity.Citizenship;
+import com.airlines.entity.CrewMember;
+import com.airlines.entity.Position;
+import com.airlines.exception.DaoOperationException;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class CrewMemberDaoImpl implements CrewMemberDao {
+
+    private final static String INSERT_CREWMEMBER_SQL = "INSERT INTO crew_members(first_name, last_name, position, birthday, citizenship) VALUES (?, ?, ?, ?, ?);";
```

Comment:
INSERT_CREWMEMBER_SQL -> INSERT_CREW_MEMBER_SQL

and so on for constants

Reply:
fixed in 45b9bb5fd29cdf8936e2dab8daa2a25ad85a7b59